{{cookiecutter.project_slug}}/Dockerfile

```diff
@@ -1,28 +1,35 @@
-FROM ubuntu:bionic
+FROM python:3.8-slim-buster as base
 
-ENV LANG C.UTF-8
-ARG DEBIAN_FRONTEND=noninteractive
-# Allow SECRET_KEY to be passed via arg so collectstatic can run during build time
-ARG SECRET_KEY
-
-# libpq-dev and python3-dev help with psycopg2
 RUN apt-get update \
-  && apt-get install -y python3.7-dev python3-pip libpq-dev curl \
-  && apt-get clean all \
+  && apt-get install -y --no-install-recommends python3.7-dev python3-pip libpq-dev gcc \
```

Comment:
This needs to be updated to `python3.8-dev`. Also check with the base image, it should already have `python3-pip` installed and may also have dev.

We need `curl` because that's how it sends logs to Heroku.

Reply:
While installing `python3.8-dev` it doesn't seem to be found
```
Couldn't find any package by glob 'python3.8-dev'
Couldn't find any package by regex 'python3.8-dev'
```

Reply:
Curl has been installed now

Reply:
This isn't resolved. We shouldn't be installing anything from python 3.7 into a 3.8 environment.

Reply:
working on it